docs/main/quickstart-guide/installation.md

```diff
@@ -81,11 +81,17 @@ helm install openebs --namespace openebs openebs/openebs --create-namespace
 The above command will install OpenEBS Local PV Hostpath, OpenEBS Local PV LVM, OpenEBS Local PV ZFS, and OpenEBS Replicated Storage components in `openebs` namespace and chart name as `openebs`.
 
 :::important
-The default OpenEBS helm chart will install both Local Storage and Replicated Storage. If you do not want to install OpenEBS Replicated Storage, use the following command:
+- The default OpenEBS helm chart will install both Local Storage and Replicated Storage. If you do not want to install OpenEBS Replicated Storage, use the following command:
 
-```
-helm install openebs --namespace openebs openebs/openebs --set engines.replicated.mayastor.enabled=false --create-namespace
-```
+  ```
+  helm install openebs --namespace openebs openebs/openebs --set engines.replicated.mayastor.enabled=false --create-namespace
+  ```
+
+- If the CustomResourceDefinitions for CSI VolumeSnapshots are already present in your cluster, you may skip their creation by using the following command:
```

Comment:
Should we also have some information somewhere, stating that the snapshot CRDs are not uninstalled? CC @niladrih

Reply:
Sure. We could add in a new docs section for 'Uninstallation'. Maybe add a footnote to the 'Installation' page with info on uninstallation. Unsure of what would be best. Also tagging @avishnu and @Abhinandan-Purkait

Reply:
We already have a [documentation for Uninstallation](https://openebs.io/docs/user-guides/uninstall). May be we can add it as an info/note here.

Reply:
Makes sense to me

Reply:
Will document this change in a separate PR.

Reply:
Documented this change in the [#PR 508](https://github.com/openebs/website/pull/508).